src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/AbstractTablesTab.java

```diff
@@ -60,13 +60,14 @@ public abstract class AbstractTablesTab implements TablesTabView.Presenter, Quer
 	public static final String TABLE_QUERY_PREFIX = "query/";
 
 	private static final String VERSION_ALERT_DRAFT_DATASET_TITLE = "This is a Draft Version of the Dataset";
-	private static final String VERSION_ALERT_DRAFT_DATASET_MESSAGE = "Administrators and Editors can edit this version and create a Stable Version for distribution. Go to the Version History to view the Stable Versions.";
+	private static final String VERSION_ALERT_DRAFT_DATASET_MESSAGE = "Administrators can edit this version and create a Stable Version for distribution. Go to the Version History to view the Stable Versions";
```

Comment:
I think we lost a change here to the message.  Expected:
Administrators **and Editors** can edit this version and create a Stable Version for distribution. Go to the Version History to view the Stable Versions.